core/src/main/scala/algebra/instances/bigDecimal.scala

```diff
@@ -12,7 +11,7 @@ trait BigDecimalInstances extends cats.kernel.instances.BigDecimalInstances {
 }
 
 class BigDecimalAlgebra(mc: MathContext) extends Field[BigDecimal] with Serializable {
-  def this() = this(MathContext.UNLIMITED)
+  def this() = this(MathContext.UNLIMITED.nn)
```

Comment:
**Classification**
Class name: java.math.MathContext.UNLIMITED
FS
.nn required: 1
Documentation: Does not mention nullability, but this is a final static field. It is initialized with the class.